### ISSUE TITLE
Add ability to run on selected filters

### ISSUE THREAD

**Issue by ghost** [plorry/VegAssist#28]
I think this will be a useful functionality to be able to expand the bot to capture different languages and localise our efforts. For example, for countries like Brazil where there is a growing vegan community and where a considerable number of people are active on Twitter. In this case, a Twitter account @vegassistente would focus on Portuguese tweets to let Portuguese speaking people reply to them.

**Comment by squeek502** [comment]
Note that the current bot retweets multiple languages, including Portuguese. See https://github.com/plorry/VegAssist/tree/master/filters for all the languages currently supported. We decided that, for now, we wouldn't split the bot by language because the other languages we're capturing seem to have very few tweets.

But anyway, about your suggestion: do you mean, for example, a key in `settings.js`? Like:

```
module.exports = {
  CREDS: {
    consumer_key:         '...',
    consumer_secret:      '...',
    access_token:         '...',
    access_token_secret:  '...'
  },
  FILTERS: [
    "portuguese"
  ]
}
```

and then if the `FILTERS` key exists, only load the filters included in the array? Doesn't sound like a bad idea and wouldn't be too hard. I'll add support for it when I get a chance.

**Comment by ghost** [comment]
Yes, that is exactly what I meant: to add an option to select a particular filter. I am happy to work on that so that I can also get more familiar with how the code works. I can send a pull request when that's done.

Let me make some tweaks to the portuguese filter. I think it could be capturing more tweets; for example, there is the genderless 'vegan' and then 'vegano' for male and 'vegana' for females. My initial thought of splitting by language is also because in Brazil, English is not widely spoken, so it is easier to see a page with only Portuguese for these people. I do understand too that if there are not enough captures, then it is not much use to separate the bot.

**Comment by squeek502** [comment]
Closed via 508307f417dd981c1a71640297442d45086fa771 / #30
